api.lua

```diff
@@ -116,3 +112,18 @@ function mail.send(...)
 		end
 	end
 end
+
+function mail.split(str, delimiter) -- flux split function
+    local rv = {}
+    local i, j = str:find(delimiter, nil, true)
+
+    while i do
+        table.insert(rv, str:sub(1, i - 1))
+        str = str:sub(j + 1)
+        i, j = str:find(delimiter, nil, true)
+    end
+
+    table.insert(rv, str)
+
+    return rv
+end
```

Comment:
Seems like this should be removed and `string.split` used instead, for usage see next comment.
```suggestion
```